README.md

```diff
@@ -103,25 +140,29 @@ Since a lot of HTTP requests are involved, it might be a cumbersome job for the
 
 ## Demo
 To demo the scanner, I make use of a vulnerable setup from [@christophetd](https://twitter.com/christophetd) using docker:
-```groovy
+```js
 docker run -p 8080:8080 ghcr.io/christophetd/log4shell-vulnerable-app
 ```
 ![image](https://user-images.githubusercontent.com/39941993/146034544-a0c0e60d-00db-44ae-823a-5e5834888108.png)
 
 Then I run the tool against the setup:
-```groovy
+```js
```

Comment:
I guess you should use shell instead of js

Reply:
@aslafy-z it just adds more color to the commands when GItHub renders the file. Its just that I love colors. :)